src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -98,6 +103,8 @@ internal PackageSourceMoniker SelectedSource
         private readonly Guid _sessionGuid = Guid.NewGuid();
         private readonly Stopwatch _sinceLastRefresh;
 
+        private bool _forceRecommender = false;
```

Comment:
default value unnecessary
```suggestion
        private bool _forceRecommender;
```

Reply:
Fixed